src/process_utility.c

```diff
@@ -1184,10 +1189,31 @@ verify_constraint_list(RangeVar *relation, List *constraint_list)
 	}
 }
 
+typedef struct HypertableIndexOptions
+{
+	bool		multitransaction;
+
+	/*
+	 * for concurrency testing purposes, it's useful to be able to pause index
```

Comment:
reformatted it a little. I think the additional comment on the latter two members are warranted, because what they do is _weird_, and only justifiable since we need it for testing.